packages/core/src/__test__/integration/two-controlled-props.test.ts

```diff
@@ -0,0 +1,129 @@
+import { createSelector } from '../../selector';
+import { createStore } from '../../store';
+import { StateConfigMap } from 'packages/core/lib/index';
+
+describe('two controlled props (pager like scenario)', () => {
+  const initialState = {
+    pageCount: 10,
+    pageIndex: 10,
+  };
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+    type State = typeof initialState;
```

Comment:
Something wrong with indents here:

```suggestion
  });
  
  type State = typeof initialState;
```